crates/milli/src/update/new/steps.rs

```diff
@@ -13,6 +13,9 @@ pub enum IndexingStep {
     ExtractingWords,
     ExtractingWordProximity,
     ExtractingEmbeddings,
+    MergingFacetCaches,
+    MergingWordCaches,
+    MergingWordProximity,
     WritingGeoPoints,
     WaitingForDatabaseWrites,
     WaitingForExtractors,
```

Comment:
Huuum, not really required in this PR but if we were using the `make_enum_progress!` macro it would handle all the derive, repr and implementation of the `IndexingStep` trait